ingests/src/main/scala/uk/ac/wellcome/platform/archive/ingests/services/IngestsWorker.scala

```diff
@@ -63,5 +65,27 @@ class IngestsWorker[CallbackDestination](
     }
   }
 
+  // The ingests monitor needs to send up to two messages:
+  //
+  //  - if the ingest is complete, it sends a message to the notifier to trigger
+  //    a callback to the user
+  //  - it sends the updated ingest to an SNS topic
+  //
+  // These two messages are independent, and so should succeed/fail independently.
+  //
+  private def sendOngoingMessages(ingest: Ingest): Try[Unit] = {
+    val callbackResult = callbackNotificationService.sendNotification(ingest)
+    val updatedIngestResult = updatedIngestsMessageSender.sendT(ingest)
+
+    (callbackResult, updatedIngestResult) match {
+      case (Success(_), Success(_))   => Success(())
+      case _ =>
```

Comment:
We can be specific about the failure using pattern matching, so we probably should be.